internal/sqlparser/testdata/valid-up/test04/input.sql

```diff
@@ -16,4 +16,27 @@ L48zPl/CFvA+KJJ6LklxfwWeVDQ+ve2OIW0B1uLhR/MsoYbDQztbgIayg6ieMO/KlQIDAQAB
 ');
 -- +goose StatementEnd
 
+INSERT INTO ssh_keys (id, "publicKey") VALUES (2, '-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
This statement is not wrapped with **-- +goose Statement**, and we validate the "-----END OPENSSH PRIVATE KEY-----" is not ignored in both cases.